### PR TITLE
support symlink development mode

### PR DESCRIPTION
**Pull request by juliandescottes**
cc @jasonLaster 

Here's a WIP to enable using symlink for launchpad w/ debugger.html

### CODE REVIEW

File: packages/devtools-launchpad/webpack.config.js [hunk 1]
```diff
@@ -48,6 +48,11 @@ module.exports = (webpackConfig, envConfig) => {
     loader: "svg-inline"
   });
 
+  // Add resolveLoader for ./node_modules to fix issues when synlinked.
+  webpackConfig.resolveLoader = webpackConfig.resolveLoader || {};
+  webpackConfig.resolveLoader.root = webpackConfig.resolveLoader.root || [];
+  webpackConfig.resolveLoader.root.push(path.resolve("./node_modules"));
```
Comment (juliandescottes) [inline packages/devtools-launchpad/webpack.config.js]: This fixes the 'cannot find module babel' errors. Added it after looking at the conversation at https://github.com/webpack/webpack/issues/784

File: packages/devtools-launchpad/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const developmentServer = require("./bin/development-server");
 const toolboxConfig = require("./webpack.config");
-const tools = require("./src/tools");
+// const tools = require("./src/tools");
```
Comment (jasonLaster) [inline packages/devtools-launchpad/index.js]: yeah... i should probably drop the async thing... i had to run `node --harmony-async-await` which is silly :)

Comment (jasonLaster) [inline packages/devtools-launchpad/index.js]: i'll drop the async and push a new version.

File: packages/devtools-launchpad/webpack.config.js (hunk 1 above)
Comment (jasonLaster) [inline packages/devtools-launchpad/webpack.config.js]: smart++

**Comment by jasonLaster** [comment]
All good changes that move us forward, I'll update the tools to be less --harmony and push

**Pull request merged**